app/main/services/process_request_json.py

```diff
@@ -13,14 +13,26 @@ def _ensure_value_list(json_string_or_list):
         return [json_string_or_list]
 
 
-def _append_conditionally(arguments, document):
+def _update_conditionally(arguments, document):
+    """
+    A transformation processor that updates field values in "target field" when
+    certain values are present in "field". The example use case is when
+    we are converting awarded, unsuccessful or cancelled brief status to closed.
+    :param arguments: dict -- the parameters to the processor as specified in configuration
+    :param document: dict -- the submitted document that we are transforming
+    """
+    _append_conditionally(arguments, document, update=True)
+
+
+def _append_conditionally(arguments, document, update=False):
```

Comment:
Sorry, I don't think this way around works - "append" is a special case of "update", but not vice versa. (i.e. here we have a function called "append" that if you pass a certain flag _doesn't_ do appending - that seems weird to me!)

Reply:
ah, i get you. You're saying that appending is a type of update. In my mind update means 'replace the whole thing'.

I'll swap around.